src/xdocs/checks/misc/outertypefilename.xml

```diff
@@ -27,35 +27,35 @@
   &lt;/module&gt;
 &lt;/module&gt;
         </source>
-        <p id="Example1-code">Example of class Test in a file named Test.java</p>
+        <p id="Example1-code">
+          Example of class Example1 in a file named Example1.java
```

Comment:
please change wording to avoid class name, we should use "file content".
`Example file content with name of file Example1.java`

please apply to all others.

Reply:
Done 😄